### ISSUE TITLE
Breadcrumbs doesn't work as expected

### ISSUE THREAD

**Issue by tedbeer** [coderaiser/cloudcmd#30]
Click breadcrumbs control on top of a panel.
The panel becomes empty.
Expected behavior - navigating to the requested level.

**Comment by coderaiser** [comment]
What browser are you use? Could you look at console in developer tools, what it shows?
You use local install of `Cloud Commander`?

**Comment by tedbeer** [comment]
I tried both - Chrome and FireFox.
CloudCommander installed via npm on Windows 7 and FreeBSD.
On localhost and on remote machine.

**Comment by coderaiser** [comment]
Could you show me call stack? `Cloud Commander` crashes with this message ` url could not be empty!`?

**Comment by tedbeer** [comment]
![cloudcmd](https://cloud.githubusercontent.com/assets/822497/7115340/1684f3fc-e1e8-11e4-9692-dc3e8e9bb00a.png)

**Comment by tedbeer** [comment]
I run release version. That's why stack trace may be not very useful.

**Comment by tedbeer** [comment]
Minification is switched off:
![cloudcmd-err](https://cloud.githubusercontent.com/assets/822497/7115604/7a0a7706-e1ea-11e4-8d02-c2a2e97442ea.png)

**Comment by coderaiser** [comment]
Error occures when `Cloud Commander` runs on Windows 7 only or on FreeBSD to?

**Comment by tedbeer** [comment]
on both systems

**Comment by coderaiser** [comment]
There was such bug. Please try to update `Cloud Commander` with:
```
npm i cloudcmd -g
```

**Comment by tedbeer** [comment]
npm i cloudcmd -g gives version 2.5.0
It's exactly the version I have reported. Bredcrumbs control doesn't work. Other functionality after trying to use breadcrumbs is also affected. It throws different exceptions.

**Comment by coderaiser** [comment]
Try to update `Cloud Commander`, it uses a lot dependencies, and this bug fixed in one of them.
Seems like it is [ponse](https://github.com/coderaiser/ponse) or something.

When you update `Cloud Commander` you update all dependencies to.
Here is my install log:
```
├── try-catch@1.0.0
├── checkup@1.0.3
├── mollify@1.0.4
├── tryrequire@1.1.5
├── jonny@1.0.1
├── pipe-io@1.1.24
├── rendy@1.0.6
├── format-io@0.9.6
├── execon@1.1.1
├── join-io@1.3.3
├── mellow@1.3.4 (win32@0.9.10)
├── minimist@1.1.1
├── files-io@1.2.5 (itype@1.0.2, extendy@1.0.1)
├── faust@1.0.1 (debug@2.1.3)
├── ponse@1.3.4 (itype@1.0.2, extendy@1.0.1, debug@2.1.3)
├── copymitter@1.6.0 (rimraf@2.2.8, findit@2.0.0, mkdirp@0.5.0, glob@4.3.5)
├── express@4.12.3 (merge-descriptors@1.0.0, utils-merge@1.0.0, cookie-signature@1.0.6, methods@1.1.1, fresh@0.2.4, cookie@0.1.2, escape-html@1.0.1, range-parser@1.0.2, finalhandler@0.3.4, content-type@1.0.1, vary@1.0.0, parseurl@1.3.0, serve-static@1.9.2, content-disposition@0.5.0, path-to-regexp@0.1.3, depd@1.0.1, on-finished@2.2.0, qs@2.4.1, debug@2.1.3, proxy-addr@1.0.7, etag@1.5.1, send@0.12.2, type-is@1.6.1, accepts@1.2.5)
├── jag@1.0.7 (tar@2.0.1, fstream@1.0.4)
├── flop@1.0.31 (timem@1.1.2, trammel@1.0.2, mkdirp@0.5.0, rimraf@2.3.2, readify@1.1.3)
├── restafary@1.2.8 (ashify@1.0.1, patchfile@1.0.1, beautifile@1.0.5)
├── minify@1.4.10 (debug@2.1.3, tomas@1.0.1, html-minifier@0.7.2, clean-css@3.1.9, uglify-js@2.4.19, css-b64-images@0.2.5)
├── http-auth@2.2.5 (node-uuid@1.4.1, coffee-script@1.8.0, htpasswd@2.1.6)
├── console-io@2.2.4 (debug@2.1.3, spawnify@1.10.8)
├── markdown-it@4.1.0 (linkify-it@1.0.0, uc.micro@1.0.0, mdurl@1.0.0, entities@1.1.1, argparse@1.0.2)
├── socket.io@1.3.5 (debug@2.1.0, has-binary-data@0.1.3, socket.io-adapter@0.3.1, engine.io@1.5.1, socket.io-parser@2.2.4, socket.io-client@1.3.5)
├── dword@1.4.2 (socket-file@1.1.0, ashify@1.0.1, patchfile@1.0.1)
└── edward@1.5.2 (socket-file@1.1.0, ashify@1.0.1, patchfile@1.0.1)
```

You should have something similar.
When did you install `Cloud Commander` last time?

**Comment by tedbeer** [comment]
My updated version is exactly the same and it does not work.
```
cloudcmd@2.5.0 D:\Users\ebespalov\AppData\Roaming\npm\node_modules\cloudcmd                                      
├── checkup@1.0.3                                                                                                
├── try-catch@1.0.0                                                                                              
├── mollify@1.0.4                                                                                                
├── tryrequire@1.1.5                                                                                             
├── jonny@1.0.1                                                                                                  
├── pipe-io@1.1.24                                                                                               
├── execon@1.1.1                                                                                                 
├── rendy@1.0.6                                                                                                  
├── format-io@0.9.6                                                                                              
├── join-io@1.3.3                                                                                                
├── minimist@1.1.1                                                                                               
├── files-io@1.2.5 (itype@1.0.2, extendy@1.0.1)                                                                  
├── mellow@1.3.4 (win32@0.9.10)                                                                                  
├── faust@1.0.1 (debug@2.1.3)                                                                                    
├── ponse@1.3.4 (itype@1.0.2, extendy@1.0.1, debug@2.1.3)                                                        
├── edward@1.5.2 (ashify@1.0.1, socket-file@1.1.0, patchfile@1.0.1)                                              
├── dword@1.4.2 (socket-file@1.1.0, ashify@1.0.1, patchfile@1.0.1)                                               
├── markdown-it@4.1.1 (linkify-it@1.0.0, mdurl@1.0.0, entities@1.1.1, uc.micro@1.0.0, argparse@1.0.2)            
├── express@4.12.3 (finalhandler@0.3.4, content-type@1.0.1, escape-html@1.0.1, parseurl@1.3.0, serve-static@1.9.2
, path-to-regexp@0.1.3, methods@1.1.1, cookie@0.1.2, utils-merge@1.0.0, cookie-signature@1.0.6, merge-descriptors
@1.0.0, fresh@0.2.4, content-disposition@0.5.0, range-parser@1.0.2, depd@1.0.1, qs@2.4.1, vary@1.0.0, debug@2.1.3
, on-finished@2.2.0, etag@1.5.1, proxy-addr@1.0.7, send@0.12.2, accepts@1.2.5, type-is@1.6.1)                    
├── copymitter@1.6.0 (rimraf@2.2.8, findit@2.0.0, mkdirp@0.5.0, glob@4.3.5)                                      
├── restafary@1.2.8 (ashify@1.0.1, patchfile@1.0.1, beautifile@1.0.5)                                            
├── flop@1.0.31 (timem@1.1.2, trammel@1.0.2, mkdirp@0.5.0, readify@1.1.3, rimraf@2.3.2)                          
├── console-io@2.2.4 (debug@2.1.3, spawnify@1.10.8)                                                              
├── jag@1.0.7 (tar@2.0.1, fstream@1.0.4)                                                                         
├── http-auth@2.2.5 (node-uuid@1.4.1, coffee-script@1.8.0, htpasswd@2.1.6)                                       
├── minify@1.4.10 (css-b64-images@0.2.5, debug@2.1.3, tomas@1.0.1, uglify-js@2.4.20, clean-css@3.1.9, html-minifi
er@0.7.2)                                                                                                        
└── socket.io@1.3.5 (debug@2.1.0, has-binary-data@0.1.3, socket.io-parser@2.2.4, socket.io-adapter@0.3.1, engine.
io@1.5.1, socket.io-client@1.3.5)                                                                                
```

**Comment by tedbeer** [comment]
I also checked out sources from the github repository and the latest dependencies. It works the same way. I mean breadcrumbs doesn't work.

**Comment by coderaiser** [comment]
Development version also didn't work?

**Comment by tedbeer** [comment]
Yes, the default version from https://github.com/coderaiser/cloudcmd.git does not work

**Comment by tedbeer** [comment]
It includes the last commit from yesterday fixing upper case extension issue.

**Comment by coderaiser** [comment]
Fixed in [restafary v1.2.9](https://github.com/coderaiser/node-restafary/releases/tag/v1.2.9).
Please, update `Cloud Commander` and give attention to installed `restafary` version.

**Comment by tedbeer** [comment]
Update restafary in sources. Now it works.
